### PR TITLE
Hack nav

### LINKED ISSUE #5485: Simplify menu
### User Story

[comment]: # (As a <user type>, I want to <task> so that <goal>.)
As a user I want to have a better and consolidated menu in all the site to help me find the areas of the site in any place.

### Why Is this Needed

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*:
Today you need to click a lot to get into explorer, and is a most needed area, each product you need to navigate in two steps away to find what you need, is not a good UX.

### Description

Create 2 menu levels for each product. 
Gitcoin/company - what, how, etc 
Bounties - Sub bounties
Hackathon - Sub Hackathon
Grants - Sub grants
Kudos - Sub kudos
Quests - Sub quests

Notifications in all pages not only internals

Example:
image

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Feature

### Current Behavior
[comment]: # (Describe what actually happened.)
Hard to navigate, and understand the sitemap 
### Expected Behavior
[comment]: # (Describe what you expected to happen.)
Better experience and discoverability 
You can move to another subarea product from everywhere/discover new products. 
### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)

### Data Requirements
[comment]: # (How will we measure the success of this feature? What kind of tracking is needed for this feature (clicks, impressions, flag)?)

### Additional Information
[comment]: # (Any other information that would be useful, content, screenshots, etc.)
image

### PR DESCRIPTION
**Pull request by octavioamu**
##### Description
This is a menu redesign for all the app, keeping the menu consistent in all places, more arguments on "why" on the issue ticket.
Also a few others fixes:
- Clicking outside dropdown will close the dropdown. 
- Design care
- Bring help links to menu
- Show the last 2 post from wordpress checks each hour if data changed. 
- Show you the product you are seeing.
- Add codefund links also as other products. 
- Add search tribes link to menu, verify profile and a few more.

image
image
image
image

##### Refers/Fixes

fix #5485
##### Testing

https://share.vidyard.com/watch/gtg81fg57HXXUkDaBtqquY?

### CODE REVIEW

**Comment by owocki** [comment]
the community subnav looks pretty thin/inconsitent with the others navs (which look awesome btw) https://docs.google.com/document/d/1q382lpfVdORIuol4NIiTYLaAQa6nQQyRqmdj5-AYRAg/edit

**Comment by owocki** [comment]
hover cutoff issues https://bits.owocki.com/yAuLEg5R

**Comment by owocki** [comment]
hmm the nav changes quite substantially as i go to the kudos or grants pages.. is this by design?

**Comment by octavioamu** [comment]
> the community subnav looks pretty thin/inconsitent with the others navs (which look awesome btw) 

Yes that is the normal dropdown but what do you think we can put there to justify a full width dropdown?

**Comment by octavioamu** [comment]
Made a slide with 3 diff messages with autoplay, about chat, tribes and leaderboard and fixed the size problem.

image

image

File: app/app/context.py [hunk 1]
```diff
@@ -36,6 +37,18 @@
...
 logger = logging.getLogger(__name__)
 
+def fetchPost(qt='2'):
+
+    url = f"https://gitcoin.co/blog/wp-json/wp/v2/posts?_fields=excerpt,title,link,jetpack_featured_media_url&per_page={qt}"
+    last_posts = requests.get(url=url).json()
+    # cache.get_or_set('last_posts', last_posts, 100)
+    # posts = {
+    #     'last_posts': last_posts
+    # }
```
Comment (thelostone-mc) [inline app/app/context.py]: ^ should we remove these ?

File: app/app/context.py [hunk 2]
```diff
@@ -36,6 +37,18 @@
...
+
+    # posts['json_context'] = json.dumps(posts)
+    print(last_posts)
```
Comment (thelostone-mc) [inline app/app/context.py]: remove print stmt

File: app/assets/v2/css/landing_page.css [hunk 3]
```diff
@@ -345,12 +345,12 @@ body.funder-page {
...
 }
 
-.navbar.following .nav-link,
+/* .navbar.following .nav-link,
 .navbar.following img,
 .navbar.following .nav-item,
 .navbar.following i.fa-angle-right {
   display: none !important;
-}
+} */
```
Comment (thelostone-mc) [inline app/assets/v2/css/landing_page.css]: ^ remove these ?

File: app/assets/v2/js/landing_page.js [hunk 4]
```diff
@@ -15,13 +15,13 @@ $(document).ready(function() {
...
     moveBackground(e);
     // Robot and Tree Parallax
-    if (!navFollowState && scrollContainer.scrollTop() > followStateHeight) {
-      $navbar.addClass('following');
-      navFollowState = true;
-    } else if (navFollowState && scrollContainer.scrollTop() < followStateHeight) {
-      $navbar.removeClass('following');
-      navFollowState = false;
-    }
+    // if (!navFollowState && scrollContainer.scrollTop() > followStateHeight) {
```
Comment (danlipert) [inline app/assets/v2/js/landing_page.js]: 👀 Should we just remove these ;)

File: app/dashboard/templates/shared/menu.html [hunk 5]
```diff
@@ -0,0 +1,307 @@
...
+              </li>
+            </ul>
+
+            <ul class="col-md-4 col-lg-2 list-unstyled {% if '/grants' in request.path %}gc-megamenu-active{% endif %}">
+              <li>
+                <a class="text-reset" href="{% url 'grants:grants' %}">
+                  <img class="mw-100 logo-static" src="{% static "v2/images/top-bar/grants_symbol_dark.svg" %}" alt="Bounties" height="48">
+                  <b class="gc-megamenu-title">{% trans "Grants" %}</b>
+                </a>
+                <p class="px-2 m-0 mb-2 text-wrap font-smaller-2">{% trans "Recurring Funding for open source" %}</p>
```
Comment (danlipert) [inline app/dashboard/templates/shared/menu.html]: Capitalize "open source"? what do you think? I notice the capitalization is a bit different on some of the items

File: app/dashboard/templates/shared/menu.html [hunk 6]
```diff
@@ -0,0 +1,307 @@
...
+                <a class="dropdown-item dropdown-item-small" href="{% url 'grants:quickstart' %}">{% trans "About Grants" %}</a>
+              </li>
+            </ul>
+            <ul class="col-md-4 col-lg-2 list-unstyled">
+              <li>
+                <a class="text-reset" href="https://codefund.app/">
+                  <img class="mw-100 logo-static" src="{% static "v2/images/top-bar/codefund_symbol_dark.svg" %}" alt="Bounties" height="48">
+                  <b class="gc-megamenu-title">{% trans "Codefund" %}</b>
+                </a>
+                <p class="px-2 m-0 mb-2 text-wrap font-smaller-2">{% trans "Ethical ads to power open source" %}</p>
```
Comment (danlipert) [inline app/dashboard/templates/shared/menu.html]: I think we should capitalize the codefund items

File: app/dashboard/templates/shared/menu.html [hunk 7]
```diff
@@ -0,0 +1,307 @@
...
+              </li>
+              <li>
+                <a class="dropdown-item dropdown-item-small" href="https://codefund.io">{% trans "About Codefund" %}</a>
+              </li>
+            </ul>
+
+            <ul class="col-md-4 col-lg-2 list-unstyled {% if '/quests' in request.path %}gc-megamenu-active{% endif %}">
+              <li>
+                <a class="text-reset" href="{% url 'quests_index' %}">
+                  <img class="mw-100 logo-static" src="{% static "v2/images/top-bar/quests_symbol_dark.svg" %}" alt="Bounties" height="48">
```
Comment (danlipert) [inline app/dashboard/templates/shared/menu.html]: alt tag here should be for quests

File: app/dashboard/templates/shared/menu.html [hunk 8]
```diff
@@ -0,0 +1,307 @@
...
+              </li>
+              <li>
+                <a class="dropdown-item dropdown-item-small" href="{% url 'funder_bounties' %}">{% trans "About Bounties" %}</a>
+              </li>
+            </ul>
+
+            <ul class="col-md-4 col-lg-2 list-unstyled {% if '/hackathon' in request.path %}gc-megamenu-active{% endif %}">
+              <li>
+                <a class="text-reset" href="{% url 'get_hackathons' %}">
+                  <img class="mw-100 logo-static" src="{% static "v2/images/top-bar/hackathons_symbol_dark.svg" %}" alt="Bounties" height="48">
```
Comment (danlipert) [inline app/dashboard/templates/shared/menu.html]: alt tag here should be hackathons

File: app/dashboard/templates/shared/menu.html [hunk 9]
```diff
@@ -0,0 +1,307 @@
...
+                  <b class="gc-megamenu-title">{% trans "Hackathons" %}</b>
+                </a>
+                <p class="px-2 m-0 mb-2 text-wrap font-smaller-2">{% trans "Build with the coolest web3 projects" %}</p>
```
Comment (danlipert) [inline app/dashboard/templates/shared/menu.html]: Wording here is a bit awkward - my suggestion: "Build projects with the coolest web3 companies"

File: app/dashboard/templates/shared/menu.html [hunk 10]
```diff
@@ -0,0 +1,307 @@
...
+              </li>
+              <li>
+                <a class="dropdown-item dropdown-item-small" href="http://hackathons.gitcoin.co/contact" target="_blank">{% trans "Get Involved" %}</a>
+              </li>
+            </ul>
+
+            <ul class="col-md-4 col-lg-2 list-unstyled {% if '/grants' in request.path %}gc-megamenu-active{% endif %}">
+              <li>
+                <a class="text-reset" href="{% url 'grants:grants' %}">
+                  <img class="mw-100 logo-static" src="{% static "v2/images/top-bar/grants_symbol_dark.svg" %}" alt="Bounties" height="48">
```
Comment (danlipert) [inline app/dashboard/templates/shared/menu.html]: alt should be grants

File: app/dashboard/templates/shared/menu.html [hunk 11]
```diff
@@ -0,0 +1,307 @@
...
+                <a class="dropdown-item dropdown-item-small" href="{% url 'grants:new' %}">{% trans "Create Grant" %}</a>
+              </li>
+              <li>
+                <a class="dropdown-item dropdown-item-small" href="{% url 'grants:quickstart' %}">{% trans "About Grants" %}</a>
+              </li>
+            </ul>
+            <ul class="col-md-4 col-lg-2 list-unstyled">
+              <li>
+                <a class="text-reset" href="https://codefund.app/">
+                  <img class="mw-100 logo-static" src="{% static "v2/images/top-bar/codefund_symbol_dark.svg" %}" alt="Bounties" height="48">
```
Comment (danlipert) [inline app/dashboard/templates/shared/menu.html]: alt should be codefund

File: app/dashboard/templates/shared/menu.html [hunk 12]
```diff
@@ -0,0 +1,307 @@
...
+              </li>
+              <li>
+                <a class="dropdown-item dropdown-item-small" href="{% url 'newquest' %}">{% trans "Create Quests" %}</a>
+              </li>
+            </ul>
+
+            <ul class="col-md-4 col-lg-2 list-unstyled {% if '/kudos' in request.path %}gc-megamenu-active{% endif %}">
+              <li>
+                <a class="text-reset" href="{% url 'kudos_main' %}">
+                  <img class="mw-100 logo-static" src="{% static "v2/images/top-bar/kudos_symbol_dark.svg" %}" alt="Bounties" height="48">
```
Comment (danlipert) [inline app/dashboard/templates/shared/menu.html]: alt tag for kudos

File: app/dashboard/templates/shared/menu.html [hunk 13]
```diff
@@ -0,0 +1,307 @@
...
+              </li>
+            </ul>
+
+            <ul class="col-md-4 col-lg-2 list-unstyled {% if '/quests' in request.path %}gc-megamenu-active{% endif %}">
+              <li>
+                <a class="text-reset" href="{% url 'quests_index' %}">
+                  <img class="mw-100 logo-static" src="{% static "v2/images/top-bar/quests_symbol_dark.svg" %}" alt="Bounties" height="48">
+                  <b class="gc-megamenu-title">{% trans "Quests" %}</b>
+                </a>
+                <p class="px-2 m-0 mb-2 text-wrap font-smaller-2">{% trans "Learn about web3 & earn rewards" %}</p>
```
Comment (danlipert) [inline app/dashboard/templates/shared/menu.html]: Capitalization here

File: app/dashboard/templates/shared/menu.html [hunk 14]
```diff
@@ -0,0 +1,307 @@
...
+              </li>
+            </ul>
+
+            <ul class="col-md-4 col-lg-2 list-unstyled {% if '/kudos' in request.path %}gc-megamenu-active{% endif %}">
+              <li>
+                <a class="text-reset" href="{% url 'kudos_main' %}">
+                  <img class="mw-100 logo-static" src="{% static "v2/images/top-bar/kudos_symbol_dark.svg" %}" alt="Bounties" height="48">
+                  <b class="gc-megamenu-title">{% trans "Kudos" %}</b>
+                </a>
+                <p class="px-2 m-0 mb-2 text-wrap font-smaller-2">{% trans "Show appreciation for each other." %}</p>
```
Comment (danlipert) [inline app/dashboard/templates/shared/menu.html]: None of the other descriptions have punctuation, so I think we should remove the `.` here

**Comment by octavioamu** [comment]
All done guys!!!!

**Pull request merged**